torchci/lib/bot/rerunGithubInfraErrorWorkflow.ts

```diff
@@ -4,10 +4,16 @@ const tagPrefix = "rerunGithubInfraFailure/";
 
 function rerunGithubInfraErrorWorkflow(app: Probot): void {
   let reruns: Map<string, number> = new Map();
+  let day = 0;
 
   // This bot is used to rerun failed workflows on pytorch/pytorch that look
   // like https://github.com/pytorch/pytorch/actions/runs/8454565307
   app.on("workflow_run.requested", async (ctx) => {
+    if (day != new Date().getDate()) {
```

Comment:
What's the timezone of getDate()?  Not a huge deal, since this defaults to rerunning the workflow anyways, but I'm wondering if the cache is likely to be cleared closer to midnight PST or 5pm PST.

Reply:
I think the docs say local timezone, so I guess where ever the bot gets deployed, probably pst if I had to guess